python/segyio/segy.py

```diff
@@ -833,6 +833,83 @@ def readonly(self):
 
         return '+' not in self._mode
 
+
+    def interpret(self, ilines, xlines, offsets=None, sorting=2):
+
+        """ (Re)interpret structure on top of a file
+
+        (Re)interprets the structure of the file based in the user supplied
+        iline, xline and offset-indecies. It is required that the number of
+        xlines, ilines and offsets correspond to the tracecount of the file.
+        Note that the actual file itself is not changed, it is only segyio's
+        interpretation of the file that changes.
+
+        Parameters
+        ----------
+
+        f : SegyFile
+        ilines : array_like
+            ilines indecies in new structure
+        xlines : array_like
+            xlines indecies in new structure
+        offset : array_like
+            offset indecies in new structure
+        sorting : int
+            Sorting in new structure
+
```

Comment:
Some spelling issues in this entire section. indecies -> indices, and I guess the description could use some reflow. 

`sorting` should be clear on accepting either an enum or maybe even a string: `f.interpret( ..., sorting = 'inline')`

Reply:
Could also clarify that `segyio.open(file) == segyio.open(ignore_geometry = True).interpret(...)`